safe-global/safe-wallet-web
# fix: Nonce form

## What it solves

Resolves #2110 

## How this PR fixes it

- Uses the previous nonce input field with queued txs dropdown and reset to recommended nonce button

## How to test it

1. Open the new tx flow
2. Observe a nonce input in the top right corner
3. Change the value
4. Observe a reset button appearing
5. Click into the input field
6. Observe a dropdown that shows all queued txs

## Screenshots
<img width="703" alt="Screenshot 2023-06-19 at 09 30 06" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/77343b76-0aea-42cb-ab2d-35ee49a6a9c8">
<img width="1052" alt="Screenshot 2023-06-19 at 09 30 17" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/11e63b16-b2f3-4519-9024-1f5cf87a1d7f">
<img width="789" alt="Screenshot 2023-06-19 at 09 30 29" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/cc874cae-0b20-4270-aaad-deb51fadf485">

## Checklist
